### PR TITLE
BDD tests environment setup

### PR DESCRIPTION
**Pull request by epapbak**
# Description

Use the `environment.py` file to prepare the dependencies in the execution context before any step is executed and run other setup/cleanup operations before the respective features.

This allows us to remove the dependency on `psql`, for example , as we now run the cleanup commands using `psycopg2` directly.

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Testing steps

Ran tests locally and within container

## Checklist
* [x] Pylint passes for Python sources
* [x] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary

### CODE REVIEW

File: features/environment.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+def before_all(context):
+    context.database_host = os.getenv("DB_HOST", "localhost")
+    context.database_port = os.getenv("DB_PORT", 5432)
+    context.database_name = os.getenv("DB_NAME", "postgres")
+    context.database_user = os.getenv("DB_USER", "postgres")
+    context.database_password = os.getenv("DB_PASS", "postgres")
+    context.kafka_hostname = os.getenv("KAFKA_HOST", "localhost")
+    context.kafka_port = os.getenv("KAFKA_PORT", "9092")
```
Comment (joselsegura) [inline features/environment.py:17]: Will this code define all those values in the context for all the tests, even if they don't use database or Kafka?

Comment (epapbak) [reply features/environment.py:17]: Yes. But I plan on adding the right filters later when all the features are tagged

**Review by tisnik — APPROVED** [review]
ok

File: features/environment.py (hunk 1 above)
Comment (joselsegura) [reply features/environment.py:17]: Well, I trust you :-)

**Review by joselsegura — APPROVED** [review]

File: features/environment.py (hunk 1 above)
Comment (epapbak) [reply features/environment.py:17]: Thanks =)

**Pull request merged**